src/config/index.ts

```diff
@@ -108,10 +108,10 @@ export default {
   },
   queueDelete: {
     queryLimit: 500,
-    itemIdChunkSize: 20,
+    itemIdChunkSize: 50,
   },
   batchDelete: {
-    deleteDelayInMilliSec: 20000,
```

Comment:
changing this from 20 to 10 to keep it consistent with account-deletion service..